script/custom-init-config.js

```diff
@@ -0,0 +1,5 @@
+// This file expects a JS object literal. 
```

Comment:
What about naming this `additional-answers-config` or something like that? It's a small nit, but there is already custom init config in some sense in the form of the `userConfig` that's passed in.